### ISSUE TITLE
Your plugin is no longer available in the Ultimaker Marketplace

### ISSUE THREAD

**Issue by VitorOM** [5axes/UniversalCuraSettings#42]
Hi, I'm not a programmer or anything like that, I've always used your plugin, it's incredible, but now it's disappeared from the store. What happened? Can I install it other than through the Ultimaker Marketplace?

**Comment by 5axes** [comment]
Due to several issues on different plugins they must be checked and modified for the 5.7 release. But I don't have time for that and as I have switch to an other slicer it's not a priority for me.

**Comment by VitorOM** [comment]
I understand, is it possible to use the plugin installed without using the Ultimaker Marketplace?

By the way, which slicer did you switch to? I see that you must be a person who is very experienced in the area.

**Comment by 5axes** [comment]
Yes you can use the curapackage on the Github. Just needs to drag and drop the file in Cura. But not tested for 5.7

**Comment by VitorOM** [comment]
> Yes you can use the curapackage on the Github. Just needs to drag and drop the file in Cura. But not tested for 5.7

which slicer did you switch to?

**Comment by 5axes** [comment]
OrcaSlicer
